### PR TITLE
opps sori

### PR DESCRIPTION
**Pull request by anhthu253**
pls review

### CODE REVIEW

**Review by PhilippWeiss85 — APPROVED** [review]
check

File: src/index.html [hunk 1]
```diff
@@ -6,11 +6,19 @@
     <title>Document</title>
   </head>
   <body>
-    <section class="card">
+    <header></header>
+    <main>
       <img
         src="https://rickandmortyapi.com/api/character/avatar/2.jpeg"
         alt="Morty Smith"
```
Comment (Mengzhen1992) [inline src/index.html]: i think there should be a p or h tag for the card's name

**Review by Mengzhen1992 — COMMENTED** [review]
pls review my comment

**Review by PhilippWeiss85 — APPROVED** [review]
looks better now

**Review by Mengzhen1992 — APPROVED** [review]
cool!

**Pull request merged**